app/src/main/java/fr/free/nrw/commons/settings/Prefs.java

```diff
@@ -7,7 +7,8 @@
     public static final String DEFAULT_LICENSE = "defaultLicense";
     public static final String UPLOADS_SHOWING = "uploadsshowing";
     public static final String MANAGED_EXIF_TAGS = "managed_exif_tags";
-    public static final String KEY_LANGUAGE_VALUE = "languageDescription";
+    public static final String KEY_DESCRIPTION_LANGUAGE_VALUE = "languageDescription";
+    public static final String KEY_APP_UI_LANGUAGE = "appUiLanguage";
```

Comment:
I am sorry to ask you to change this again, but would you mind making it:
```
public static final String DESCRIPTION_LANGUAGE = "descriptionLanguage";
public static final String APP_UI_LANGUAGE = "appUiLanguage";
```

Reply:
No problem, I will do that.